crates/notion-core/src/serial/manifest.rs

```diff
@@ -26,6 +26,7 @@ pub struct Manifest {
 pub struct NotionManifest {
     pub node: String,
     pub yarn: Option<String>,
+    pub events_plugin: Option<String>,
```

Comment:
This shouldn't live in the manifest, but in the Notion config file. We can fix this up in a follow-up PR though.

Reply:
ok, added a note about this too